app/views/comments/_comment.html.erb

```diff
@@ -124,7 +124,7 @@ class="comment <%= comment.current_vote ? (comment.current_vote[:vote] == 1 ?
           <% if comment.flags > 0 &&
           comment.show_score_to_user?(@user) &&
           (comment.user_id == @user.try(:id) || @user.try("is_moderator?")) %>
-            | <%= comment.vote_summary_for_user(@user).downcase %>
+            <%= "| #{comment.vote_summary_for_user(@user).downcase}" if comment.vote_summary_for_user(@user) %>
```

Comment:
```suggestion
<% if (s = comment.vote_summary_for_user(@user).downcase).present? %>
  | <%= s %>
<% end %>
```

Something like this would memoize the call.

Reply:
done.